src/sections/Header/header.scss

```diff
@@ -82,4 +82,57 @@
             margin: 0 1.75rem;
         }
     }
+}
+
+.header__nav__mobile{
+    display: none;
+}
+
+@media only screen   
+and (min-device-width : 320px)   
+and (max-device-width : 480px) {
+    .header .header__nav ,.header.opaque .header__nav{
+       display: none;
+    }
+    .header__nav__mobile{
+        display: block;
+    }
+    .bm-burger-bars{
+        background: $color-black;
+        width: 10%;
+        height: 3%!important;
+        right: 8% !important;
+        left: auto!important;
+        &:nth-child(1){
```

Comment:
Little unsure of why we have to hard code children and also override top with !important. Is there a better, and less rigid way to do this?

Reply:
For now will go with this style override approach